core/vm/jump_table.go

```diff
@@ -91,24 +92,29 @@ func newVerkleInstructionSet() JumpTable {
 	return validate(instructionSet)
 }
 
-func NewPragueEOFInstructionSetForTesting() JumpTable {
-	return newPragueEOFInstructionSet()
+func NewEOFInstructionSetForTesting() JumpTable {
+	return newEOFInstructionSetForTesting()
```

Comment:
Why do we have an exported and unexported version of the same function?

Reply:
I assume because we added the testing function just to do fuzz testing, but didn't want to churn core geth code when we decide to delete it.

Reply:
Happy to remove to consolidate it into the exported function if that's better. Just trying to retain what was there.

Reply:
Exported is named "ForTesting" to signal that nobody should rely on it as official api. Fuzzing yes, and maybe also goevmlab